packages/common/src/store/collectibles/reducer.ts

```diff
@@ -0,0 +1,33 @@
+import { createReducer, ActionType } from 'typesafe-actions'
+
+import { Collectible, ID } from 'models'
+
+import * as actions from './actions'
+import { UPDATE_USER_COLLECTIBLES } from './actions'
+
+type CollectiblesActions = ActionType<typeof actions>
+
+type CollectiblesState = {
+  collectibles: { [id: ID]: Collectible[] }
+}
+
+const initialState = {
+  collectibles: {}
+}
+
+const collectiblesReducer = createReducer<
```

Comment:
Will do a more thorough review soon, but why are we using createReducer instead of redux-toolkit `create-slice` here?